.circleci/config.yml

```diff
@@ -1734,6 +1734,31 @@ jobs:
     - store_test_results:
         path: bazel-testlogs
 
+  pytorch_doc_test:
+    environment:
+      BUILD_ENVIRONMENT: pytorch-doc-test
+      # TODO: stop hardcoding this
+      DOCKER_IMAGE: "308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3.6-gcc5.4:8fcf46ef-4a34-480b-a8ee-b0a30a4d3e59"
+    resource_class: large
```

Comment:
Perhaps as separate CI, but do we really need large machine to build docs?

Reply:
nope, this was copypasta lol